### ISSUE TITLE
Organize data directory

### ISSUE THREAD

**Issue by jotok** [DCActionforChildren/dcaction#159]
I think we should create subdirectories for scripts, input files, output files, and configuration files (maybe). Input files could be split into data files (which need to be processed to feed into the app) and geo files. What do you think, @cmgiven ?

I'm thinking something like:

```
data/
  - scripts/
  - geodata/
  - inputs/
  - outputs/
```

**Comment by cmgiven** [comment]
Agreed, this would be a great step. Per our conversation, here are the files the front-end reads:
- `charters.json`
- `dcps.json`
- `hospitals.json`
- `libraries.json`
- `neighborhoods.csv`
- `neighborhoods44.json`
- `source.csv`
